src/components/IconMenu.tsx

```diff
@@ -0,0 +1,23 @@
+import type { MenuProps } from '@chakra-ui/react';
+import { IconButton, Menu, MenuButton, MenuList } from '@chakra-ui/react';
+import type { FunctionComponent } from 'react';
+
+export type ItemMenuProps = MenuProps & {
+  iconButtonVariant?: 'outline' | 'ghost' | 'solid' | 'link' | 'unstyled';
+  icon: JSX.Element;
+  children?: React.ReactNode;
```

Comment:
Please import from `react`.
```suggestion
  children?: ReactNode;
```